src/pages/downloads.tsx

```diff
@@ -440,22 +428,13 @@ const DownloadsPage: NextPage<Props> = ({ data }) => {
             <Flex
               sx={{ mt: '0 !important' }}
               flexDirection={{ base: 'column', md: 'row' }}
-              justifyContent='space-between'
+              justifyContent='flex-end'
               alignItems='center'
             >
-              <Stack p={4} display={{ base: 'none', md: 'block' }} mx='auto'>
-                <Text textStyle='hero-text-small'>
-                  {totalDevBuilds > 0
-                    ? `Showing ${Math.min(amountDevBuilds, totalDevBuilds)} latest releases of
-                    a total ${totalDevBuilds} releases`
-                    : `No releases`}
-                </Text>
-              </Stack>
-
               {totalDevBuilds > amountDevBuilds && (
                 <Stack
                   sx={{ mt: '0 !important' }}
-                  borderLeft={{ base: 'none', md: '2px solid #11866f' }}
+                  borderLeft={{ base: 'none', md: '2px solid #06fece' }}
```

Comment:
```suggestion
                  borderLeft={{ base: 'none', md: '2px solid var(--chakra-colors-green-200)' }}
```

Reply:
Actually:
<img width="459" alt="image" src="https://user-images.githubusercontent.com/54227730/205409064-24bd9ed7-eb05-4dcf-98ad-417c207a7757.png">

This needs to be light/dark mode responsive... we should be using `primary` here
```suggestion
                  borderLeft={{ base: 'none', md: '2px solid var(--chakra-colors-primary)' }}
```